examples/ecs/command_error_handling.rs

```diff
@@ -0,0 +1,74 @@
+use bevy::{core::FixedTimestep, prelude::*};
+
+fn main() {
+    App::build()
+        .add_startup_system(setup.system())
+        .add_system_set(
+            SystemSet::new()
+                .with_run_criteria(FixedTimestep::step(0.5))
+                .with_system(remove_components.system()),
+        )
+        .add_system_set(
+            SystemSet::new()
+                .with_run_criteria(FixedTimestep::step(2.0))
+                .with_system(despawn_all_entities.system()),
+        )
+        .run();
+}
+
+struct A(usize);
+
+#[derive(Bundle, Default)]
+struct B {
+    value: usize,
+}
+
+struct FailedDespawnAttempts(usize);
+
+fn setup(mut commands: Commands) {
+    for i in 0..3 {
+        // Note that `insert` and `insert_bundle` are fallible functions.
+        // If no error handler is specified, the default behavior is to log the error, and continue.
+        // These calls to `insert` and `insert_bundle` will not fail, since the entity is valid.
+        commands.spawn().insert(A(i)).insert_bundle(B::default());
+    }
+
+    commands.insert_resource(FailedDespawnAttempts(0));
```

Comment:
I would almost always initialize this as a resource directly in the AppBuilder; Commands to add resources are unnecessarily prone to introducing bugs due to their delayed effects.